luminoso_api/v5_download.py

```diff
@@ -32,14 +32,15 @@ def iterate_docs(client, expanded=False, progress=False):
     URL points to a project.
 
     If expanded=True, it will include additional fields that Luminoso added in
-    its analysis, such as 'tokens' and 'vector'.
+    its analysis, such as 'terms' and 'vector'.
 
     Otherwise, it will contain only the fields necessary to reconstruct the
     document: 'title', 'text', and 'metadata'.
 
     Shows a progress bar if progress=True.
     """
-    num_docs = client.get('docs', limit=1)['total_count']
+    # Get total number of docs from the project record
+    num_docs = client.get()['counts']['__all__']
```

Comment:
`['counts']['__all__']` is a v4 API return format.  For v5, you want `['document_count']`.